src/main/resources/assets/appliedenergistics2/lang/en_US.lang

```diff
@@ -330,7 +330,7 @@ gui.tooltips.appliedenergistics2.SchedulingModeRandom=Export items in random mod
 gui.tooltips.appliedenergistics2.ItemsStored=Items Stored: %s
 gui.tooltips.appliedenergistics2.ItemsRequestable=Items Requestable: %s
 gui.tooltips.appliedenergistics2.P2PFrequency=Frequency: %s
-gui.tooltips.appliedenergistics2.OreFilterHint=Filter using oredictionary names
+gui.tooltips.appliedenergistics2.OreFilterHint=Filter items using ore dictionary names
 gui.tooltips.appliedenergistics2.OreFilter=Oredictioary filter
```

Comment:
typo in line 334: needs to be `Ore Dictionary Filter`

Reply:
https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/commit/4e9520fe1da423a02a61d0cf9562f25c6240aad0